product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Optimizing subscriber-only groups
+description: 'Optimizing subscriber-only groups in EDB Postgres Distributed'
+---
+
+While many use cases rely on accessing a database node which can handle queries and updates, there are also use cases which only require access to a node that can handle read-only database queries.
+Read scaling like this, by moving the read-only traffic away from active database nodes in the cluster, can improve the performance of the core cluster, whilst making database access more widely available.
+
+PGD provides this read scaling functionality through subscriber-only nodes and groups.
+
+With PGD 5.6 and later, this topology can be optimized. For clusters using groups for their data nodes, subscriber only groups can use a more efficient model which uses a group leader, similar to a write leader in PGD Proxies.
+
+Each subscriber-only group then has a node which replicates changes to the other subscriber-only nodes in its group.
+You can have multiple subscriber-only groups, which can address different regions or different application demands.
+
+This group leader acts as a replication proxy for incoming changes.
+
+This model cannot be enabled if a cluster has any of the following:
```

Comment:
Will a positive tone be better? Like say Following are pre-requisites to enable this model.